### PR TITLE
Allow overwriting trust policy actions

### PR DESCRIPTION
**Pull request by allistera**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -36,6 +36,15 @@ variable "assume_role_conditions" {
   default     = []
 }
 
+variable "assume_role_actions" {
+  type = set(object({
+    type        = string
+    identifiers = list(string)
+  }))
```
Comment (mariux) [inline variables.tf:43]: This should be set/list of string as we don't use complex objects here, like we did in conditions or principals:

```suggestion
  type = set(string)
```

**Review by mariux — CHANGES_REQUESTED** [review]
hi @allistera, thanks for your awesome contribution. 

If I am not mistaken, I spotted one small issue in the type of the new variable.

**Comment by allistera** [comment]
Hi @mariux  yes good spot! Serves me right for doing a quick copy and paste job :)! I have updated the PR. Thanks

**Review by mariux — CHANGES_REQUESTED** [review]
my bad, missed something in my last suggestion

**Comment by allistera** [comment]
Updated. Thanks again

**Review by mariux — APPROVED** [review]

**Comment by mariux** [comment]
thanks again for the contribution, we will run tests and merge to master and asap released. ;)

**Pull request merged**